Sources/InjectionNext/InjectionNext.swift

```diff
@@ -148,6 +148,18 @@ open class InjectionNext: SimpleSocket {
                 let dylib = NSTemporaryDirectory() + dylibName
                 try! data.write(to: URL(fileURLWithPath: dylib))
                 injectAndSweep(dylib)
+            case .metrics:
```

Comment:
The idea of using a posting notification with userInfo for people to tap into is very interesting. Perhaps the notification name could be read out of the message so the functionality could be extended later on without having to change the InjectionCommand protocol.